server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -0,0 +1,418 @@
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            int[] versions = modelDispatcher.listModelVersions(modelId);
+            versions = versions == null ? new int[0] : versions;
+
+            Response response = Utils.ok(new ModelVersionListResponse(versions));
+            LOG.info("List {} versions of model {}", versions.length, modelId);
+            return response;
+          });
+
+    } catch (Exception e) {
+      return ExceptionHandlers.handleModelException(OperationType.LIST_VERSIONS, model, schema, e);
+    }
+  }
+
+  @GET
+  @Path("{model}/versions/{version}")
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "get-model-version." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "get-model-version", absolute = true)
+  public Response getModelVersion(
+      @PathParam("metalake") String metalake,
+      @PathParam("catalog") String catalog,
+      @PathParam("schema") String schema,
+      @PathParam("model") String model,
+      @PathParam("version") int version) {
+    LOG.info(
+        "Received get model version request: {}.{}.{}.{}.{}",
+        metalake,
+        catalog,
+        schema,
+        model,
+        version);
+    NameIdentifier modelId = NameIdentifierUtil.ofModel(metalake, catalog, schema, model);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            ModelVersion mv = modelDispatcher.getModelVersion(modelId, version);
+            Response response = Utils.ok(new ModelVersionResponse(DTOConverters.toDTO(mv)));
+            LOG.info("Model version got: {}.{}", modelId, version);
+            return response;
+          });
+
+    } catch (Exception e) {
+      return ExceptionHandlers.handleModelException(
+          OperationType.GET, versionString(model, version), schema, e);
+    }
+  }
+
+  @GET
+  @Path("{model}/aliases/{alias}")
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "get-model-alias." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "get-model-alias", absolute = true)
+  public Response getModelVersionAlias(
+      @PathParam("metalake") String metalake,
+      @PathParam("catalog") String catalog,
+      @PathParam("schema") String schema,
+      @PathParam("model") String model,
+      @PathParam("alias") String alias) {
+    LOG.info(
+        "Received get model alias request: {}.{}.{}.{}.{}",
+        metalake,
+        catalog,
+        schema,
+        model,
+        alias);
+    NameIdentifier modelId = NameIdentifierUtil.ofModel(metalake, catalog, schema, model);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            ModelVersion mv = modelDispatcher.getModelVersion(modelId, alias);
+            Response response = Utils.ok(new ModelVersionResponse(DTOConverters.toDTO(mv)));
+            LOG.info("Model alias got: {}.{}", modelId, alias);
```

Comment:
Logs like this can be turned to the debug level.

Reply:
It follows the convention to add all the info log before and after the operation, you can check all the other operations, they all have the info logs.